app/src/ui/merge-conflicts/unmerged-file.tsx

```diff
@@ -0,0 +1,332 @@
+  readonly dispatcher: Dispatcher
+}> = props => {
+  const onDropdownClick = makeManualConflictDropdownClickHandler(
+    props.path,
+    props.repository,
+    props.dispatcher
+  )
+
+  const content = (
+    <>
+      <div className="column-left">
+        <PathText path={props.path} availableWidth={200} />
+        <div className="file-conflicts-status">{manualConflictString}</div>
+      </div>
+      <div className="action-buttons">
+        <Button
+          className="small-button button-group-item resolve-arrow-menu"
+          onClick={onDropdownClick}
+        >
+          Resolve
+          <Octicon symbol={OcticonSymbol.triangleDown} />
+        </Button>
+      </div>
+    </>
+  )
+
+  return renderConflictedFileWrapper(props.path, content)
+}
+
+function renderConflictedFileWrapper(
+  path: string,
+  content: JSX.Element
+): JSX.Element {
+  return (
+    <li key={path} className="unmerged-file-status-conflicts">
+      <Octicon symbol={OcticonSymbol.fileCode} className="file-octicon" />
+      {content}
+    </li>
+  )
+}
+
+const renderConflictedFileWithConflictMarkers: React.SFC<{
+  readonly path: string
+  readonly status: ConflictsWithMarkers
+  readonly resolvedExternalEditor: string | null
+  readonly onOpenEditorClick: () => void
+  readonly repository: Repository
+  readonly dispatcher: Dispatcher
+}> = props => {
+  const humanReadableConflicts = calculateConflicts(
+    props.status.conflictMarkerCount
+  )
+  const message =
+    humanReadableConflicts === 1
+      ? `1 conflict`
+      : `${humanReadableConflicts} conflicts`
+
+  const disabled = props.resolvedExternalEditor === null
+  const tooltip = editorButtonTooltip(props.resolvedExternalEditor)
+  const onDropdownClick = makeMarkerConflictDropdownClickHandler(
+    props.path,
+    props.repository.path,
+    props.dispatcher
+  )
+
+  const content = (
+    <>
+      <div className="column-left">
+        <PathText path={props.path} availableWidth={200} />
+        <div className="file-conflicts-status">{message}</div>
+      </div>
+      <div className="action-buttons">
+        <Button
+          onClick={props.onOpenEditorClick}
+          disabled={disabled}
+          tooltip={tooltip}
+          className="small-button button-group-item"
+        >
+          {editorButtonString(props.resolvedExternalEditor)}
+        </Button>
+        <Button
+          onClick={onDropdownClick}
+          className="small-button button-group-item arrow-menu"
+        >
+          <Octicon symbol={OcticonSymbol.triangleDown} />
+        </Button>
+      </div>
+    </>
+  )
+  return renderConflictedFileWrapper(props.path, content)
+}
+
+/** makes a click handling function for manual conflict resolution options */
+const makeManualConflictDropdownClickHandler = (
+  relativeFilePath: string,
+  repository: Repository,
+  dispatcher: Dispatcher
+) => {
+  return () => {
+    const items: IMenuItem[] = [
+      {
+        label: 'Use our version',
```

Comment:
As I was looking to the screenshots you provided @outofambit I'm not sure what you exactly mean with `Use our version`. Maybe we can find something more descriptive here?

Reply:
Thanks @agisilaos! I think that's captured in https://github.com/desktop/desktop/issues/6782 and @shiftkey's comment above, where the intent is to use more descriptive verbiage to indicate which version it is (in other words, what branch it came from and whether it was added, modified, or deleted). Please correct me if I'm wrong @outofambit.